packages/block-library/src/post-featured-image/index.php

```diff
@@ -31,7 +31,11 @@ function render_block_core_post_featured_image( $attributes, $content, $block )
 	$overlay_markup = get_block_core_post_featured_image_overlay_element_markup( $attributes );
 
 	if ( $is_link ) {
-		$attr['alt'] = trim( strip_tags( get_the_title( $post_ID ) ) );
+		if( get_the_title( $post_ID ) ) {
```

Comment:
Small spacing issue on this if.

```suggestion
		if ( get_the_title( $post_ID ) ) {
```